### PR TITLE
User example

### PR DESCRIPTION
**Pull request by anhhuy0501**

### CODE REVIEW

File: deploy.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```
Comment (thala321) [inline deploy.sh:1]: Remove this file

File: plugin/plugin/tests/tests.rs [hunk 2]
```diff
@@ -8,7 +8,7 @@ use sp_runtime::Digest;
 use std::str::FromStr;
 use structmap::GenericMap;
 
-const LIBPATH: &'static str = "../../target/debug/libtest_plugin.dylib";
+const LIBPATH: &'static str = "../../target/debug/libtest_plugin.so";
```
Comment (thala321) [inline plugin/plugin/tests/tests.rs:11]: Note: this will only support Ubuntu

File: user-example/solana/two-tables/src/lib.rs [hunk 3]
```diff
@@ -0,0 +1,83 @@
+// mod mapping;
```
Comment (thala321) [inline user-example/solana/two-tables/src/lib.rs:1]: Remove these comments

File: user-example/solana/two-tables/src/mapping.rs [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    let transaction_ts = Transaction {
+        id: transaction_id,
+        signature: format!("{:?}", transaction.transaction.transaction.signatures),
+        timestamp: Default::default(),
+        fee: Default::default(),
+        block: block_id.clone(),
+        block_number: Default::default(),
+        success: true,
+
+        // pub id: String,
```
Comment (thala321) [inline user-example/solana/two-tables/src/mapping.rs:24]: Remove these comments

File: user-example/solana/two-tables/src/models.rs [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+#[derive(Default, Clone, ToMap)]
+pub struct Block {
+    pub id: String,
+    pub block_number: i64,
+    pub block_hash: String,
+    pub sum_fee: i64,
+    pub transaction_number: i64,
+    pub success_rate: i64
+
+    // id: ID!
```
Comment (thala321) [inline user-example/solana/two-tables/src/models.rs:14]: Remove these comments

File: user-example/solana/two-tables/src/project.yaml [hunk 6]
```diff
@@ -0,0 +1,7 @@
+schema:
+  file: ./schema.graphql
+
+dataSources:
+  - kind: solana
+    name: Index
```
Comment (thala321) [inline user-example/solana/two-tables/src/project.yaml:6]: Please change the index name for every examples

File: user-example/solana/two-tables/src/lib.rs (hunk 3 above)
Comment (anhhuy0501) [reply user-example/solana/two-tables/src/lib.rs:1]: done

**Comment by thala321** [comment]
LGTM

**Review by thala321 — APPROVED** [review]
LGTM

**Pull request merged**